buildenv/jenkins/JenkinsfileBase

```diff
@@ -11,6 +11,55 @@ def getBuildList() {
 	def TESTPROJECT = TESTPROJECTS[simpleTarget]
 	return TESTPROJECT
 }
+
+def getJCK(){
+	if(!fileExists("$WORKSPACE/jck/${env.JCK_VERSION}/")){
```

Comment:
This check is redundant. I personal think checking should be caller's functionality.

Reply:
updated